CHANGELOG.md

```diff
@@ -26,6 +26,7 @@ _None_
 
 - Add `tools:ignore="InconsistentArrays"` to `available_languages.xml` to avoid a linter warning on repos hosting multiple app flavors. [#390]
 - Add the ability to provide a custom message for builds triggered via `buildkite_trigger_build` action [#392]
+- Add optional `release_notes_file_path` to `ios_update_release_notes` and `android_update_release_notes` [#396]
```

Comment:
@leandroalonso Oh shoot I just realized that I do failed to spot that this release notes entry was in the wrong place, it should have been under the `trunk` section.

As a result, when @oguzkocer  did a new release a couple of minutes ago in #398, our release script didn't see that this was a new entry under "New Features" when it tried to determine the next version to bump to according to SemVer.

That is not dramatic because the version that got released luckily was also to ship a "New Feature" (ie minor version bump according to semver) so we ended up with the correct version number `5.3.0` being released after all.

Still, for consistency, it would be nice to fix that, by moving this entry from `5.2.0` to `5.3.0`, and also update the description of the `5.3.0` GitHub Release accordingly; even if that would be fixed in `trunk` only after the fact and release, better fix the inconsistency for the future.

Reply:
Fixed in #399.

Reply:
Thank you so much @oguzkocer!